### ISSUE TITLE
Is this continuation of cloudius-systems/capstan?

### ISSUE THREAD

**Issue by javor** [mikelangelo-project/capstan#10]
Is this continuation of cloudius-systems/capstan repo?

**Comment by gberginc** [comment]
@javor Yes. It's part of a [project](http://mikelangelo-project.eu) in which Cloudius (now ScyllaDB) is also involved, however they are mainly focusing on improving OSv itself. Since there is little activity in upstream repository it hasn't been decided whether this repo should become the main version or we'll try to merge it with the cloudius-systems/capstan.

If you are interested in the project, you can check high [level description](https://www.mikelangelo-project.eu/technology/unikernel-application-management/) and the slightly outdated [manual](https://www.mikelangelo-project.eu/technology/unikernel-application-management-users-manual/)

We are currently preparing for a release of all the new changes we've done. I expect it to be ready end of this week or early next, along with the fully updated manual.

**Comment by javor** [comment]
Great. Probably I can try to contribute to this project.

**Comment by gberginc** [comment]
Sounds great! We are open for discussion if you'd like to check whether your ideas are already plenned.

If you are interested, we are also working on providing the new capabilities for composing OSv-based images into [UniK](https://github.com/emc-advanced-dev/unik) project. This  will then allow for unified management of unikernels.

**Comment by javor** [comment]
Hello again. I have simple question. Have you got any Jira like service and how to contribute to this project?

**Comment by gberginc** [comment]
@javor Apologies for not responding sooner. We are currently not using external Jira; we are considering using pivotal tracker, but since we have just an internal team, we are managing tasks internally. For code reviews we are using internal Gerrit.

For external contributors, I suggest you propose the enhancement/fix via GitHub issue for now. Proposing this via issue will allow you to discuss ideas with us and making sure we'll be able to accept the changes. Patches should be provided as github pull requests; this will also help us transition to github PRs even for our internal work (have been discussing but there was no particular need for this yet).
